src/internal/hooks/use-visual-mode/__tests__/use-visual-mode.test.tsx

```diff
@@ -38,4 +44,25 @@ describe('useVisualRefresh', () => {
     expect(console.warn).toHaveBeenCalledWith(expect.stringMatching(/Dynamic visual refresh change detected/));
     expect(screen.getByTestId('current-mode')).toHaveTextContent('false');
   });
+
+  describe('Window Symbol isVisualRefresh ', () => {
+    afterEach(() => {
+      window[Symbol.for('isVisualRefresh')] = undefined;
+    });
+
+    test('should return true when Window Symbol isVisualRefresh is true', () => {
```

Comment:
Should we also add a test when the symbol returns `false` value?

Reply:
Also, what about the test when `awsui-visual-refresh` class already exists, but symbol is not defined or returns false?